packages/dd-trace/src/opentracing/span.js

```diff
@@ -262,6 +263,10 @@ class DatadogSpan {
           .padStart(8, '0')
           .padEnd(16, '0')
       }
+
+      if (fields.apmTracingEnabled === false) {
```

Comment:
another option could be to create a new module, initialize it with the config and if `apmTracingEnabled` is false then subscribe to `'dd-trace:span:start'` event, check if the span has no parent and if so, set the `_dd.apm.enabled` tag.

Reply:
If I understand this correctly, all the spans are going to have tag with `APM_TRACING_ENABLED=0` when `apmTracingEnabled === false`, not only the rootspans, am I right? Is the expected behaviour?

Reply:
it's added only when there is no context and parent so only in root spans

Reply:
I think that we can't work correctly with this approach, `tracer.startSpan` is called from more points in the code([for example](https://github.com/DataDog/dd-trace-js/blob/master/packages/datadog-plugin-aws-sdk/src/base.js#L47)). If in those points they creates a new rootSpan because they are not already in one, they are going to go without `APM_TRACING_ENABLED=0`.

Reply:
My fault, calling to `tracer.startSpan` is ok.